packages/template-retail-react-app/app/pages/product-list/index.jsx

```diff
@@ -148,6 +148,16 @@ const ProductList = (props) => {
         {
             parameters: {
                 ...restOfParams,
+                perPricebook: true,
+                allVariationProperties: true,
+                expand: [
+                    'availability',
+                    'prices',
+                    'represented_products',
+                    'variations',
+                    'custom_properties',
+                    'images'
+                ],
```

Comment:
This is a very good point that @joeluong-sfcc is making here. One thing to note tho, is if we make this change to provide a bunch of data, pulling back in the future to reduce that data set would be a breaking change. Maybe I can get both of your opinions on this.

Reply:
I think we should pull the necessary data to get strikethrough prices here, in the future if we need to pull more, we only add more instead of reducing the data, then it wouldn't be a breaking change